### ISSUE TITLE
Unable to connect to dydx dapp via wallet connect

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#21204]
**Steps:** 

1. Go to https://trade.dydx.exchange/portfolio/overview?utm_source=dydx-website 
2. Generate wallet connect QR 
3. Scan wallet QR via Status app (try both Android and IOS because the actual result is different)
4. See if you will be able to proceed with connection

**Actual result:** 

**Android:**  nothing happens when scanning QR. Connection is not triggered

https://github.com/user-attachments/assets/00b03085-d25d-44e0-9b7b-9862632dc81a

**IOS:** QR can be successfully scanned, connection process is triggered but stucks on Dapp's side. Dapp is unable to send event to Status app which needs to be signed for connection process to be completed.  

### Additional Information

- Status version: nightly 
- Operating System: Android 12, iOS 17.6.1

**Comment by mohsen-ghafouri** [comment]
I spend some time on it, everything from our side looks correct, but it looks app doesn't handle event correctly. will unassign if someone else can investigate more.

**Comment by churik** [comment]
moved to the next milestone due to limited capacity

**Comment by churik** [comment]
@clauxx if there is no obvious way to deal with this sort of issue, we likely might close it as "Not planned", as it seems to be third-party issue

**Comment by clauxx** [comment]
> [@clauxx](https://github.com/clauxx) if there is no obvious way to deal with this sort of issue, we likely might close it as "Not planned", as it seems to be third-party issue

@churik i'm 99% sure the issue is in the color-inverted (dark) qr code. From what I tested, on android the camera library we use doesn't pick them up, so most likely it's from our side.

**Comment by shivekkhurana** [comment]
I'm removing the 2.33 milestone here. Will add it to 2.34.

**Comment by clauxx** [comment]
Seems like dydx no longer shows the inverted qr codes, so maybe we can re-name this issue to fix support for dark/inverted qr codes? I have a [branch](https://github.com/status-im/status-mobile/tree/cl-21204-dark-qr-scan) where i started the migration to `react-native-vision-camera`, but it seems like the scope of changes is larger than expected, since features like zooming are not supported by default and should be implemented with custom gestures.

**Comment by shivekkhurana** [comment]
Do we even need this ? Given dydx is not a problem anymore. 

And inverted QR codes are not the norm. I'm inclined to close this issue.

**Comment by clauxx** [comment]
> Do we even need this ? Given dydx is not a problem anymore.
> 
> And inverted QR codes are not the norm. I'm inclined to close this issue.

Yeah can be closed I guess. If inverted QRs become an issue later on, we can re-open it.
